content/v2.0/influx-templates/create.md

```diff
@@ -1,21 +1,105 @@
 ---
-title: Create an Influx Template
+title: Create an Influx template
 description: >
-  placeholder
+  Use the InfluxDB UI and the `influx pkg export` command to create Influx templates.
 menu:
   v2_0:
     parent: Influx templates
     name: Create a template
-    identifier: Create an Influx Template
+    identifier: Create an Influx template
 weight: 101
 v2.0/tags: [influx templates]
 ---
 
-## Placeholder outline
-- Create a template
-  - `influx pkg`
-  - Package everything vs package specific things
-  - Include secrets (only keys are included, uses will still need to populate the secret)
-  - Submit a template to the InfluxDB community template repo
-  - View your package summary
-  - Validate a package
+Use the InfluxDB user interface (UI) and the `influx pkg export` command to
+create Influx templates.
+Add buckets, Telegraf configurations, tasks, and more in the InfluxDB UI and export
+them as a template.
+
+{{% note %}}
+The simplest way to create a template is to create a new organization and build
+the template within the organization.
+**InfluxDB OSS** supports multiple organizations.
+In **InfluxDB Cloud**, your user account is an organization.
```

Comment:
we may want to revisit this when @desa 's work with sample data gets in. That could align with pkgs to create pkgs with sample data and see real results of what the pkg may look like. 

While I'm on the subject, @russorat I have ideas of generating a workflow for this that is programmaatic, based on an openapi spec for the entire pkger resource collection. Food for thought, a workflow that requires the UI know next to nothing except how to use an open api spec 👌 and glue together with sample data 🤔.
cc @jademcgough

Reply:
the work to make the open api specs for the resources can also be used by kubernetes with the CRDs as well